index.html

```diff
@@ -0,0 +1,171 @@
+    </section>
+    <!-- main program section -->
+
+    <main>
+        <div class="main-prog">
+            <h3>Main Program</h3>
+            <hr>
+        </div>
+        <div class="Cards">
+            <div class="card">
+                <img src="./images/icon-1.png" alt="icon">
+                <h4>Lecture</h4><h6>Listen to the speakers from various countries about the messages of sharing and opening.</h6>
+            </div>
+            <div class="card">
+                <img src="./images/icon-2.png" alt="icon">
+                <h4>CC Exhibition</h4><h6>Appreciate various creations applying CC license of artists, organized from Art Center Nabi</6>
+            </div>
+            <div class="card">
+                <img src="./images/icon-3.png" alt="icon">
+                <h4>Forum</h4><h6>Have the time to share your thoughts and opinions with experts for each topic.</6>
+            </div>
+            <div class="card">
+                <img src="./images/icon-4.png" alt="icon">
+                <h4>Workshop</h4><h6>Try creating your own work using open source license rather than just watching at it.</h6>
+            </div>
+            <div class="card">
+                <img src="./images/icon-5.png" alt="icon">
+                <h4>CC Ignite</h4><h6>Get opportunities to network with CC affiliates around the world, also after the summit.</6>
+            </div>
+        </div>
+        <div class="Button">
+            <button>Join Global Summit 2023</button>
+            <a href="#">SEE THE WHOLE PROGRAM</a>
+        </div>
+    </main>
+
+    <!-- featured speakers -->
+
+    <section class="presenter" id="speak">
+        <div class="pres-head">
+            <h3>Featured Speakers</h3>
+            <hr>
+        </div>
+        <div class="spcards" id="sp-cards">
+            <!-- load data from js file -->
+
+            <div class="sp-button" id="more-button">
+                <button>MORE <img src="./images/arrow-down.png" alt="Down icon"></button>
+            </div>
+        </div>
+    </section>
+
+    <!--sponsors section -->
+
+    <section class="partners">
+        <div class="part-body">
+            <div class="part-head">
+                <h3>PARTNERS</h3>
+                <hr>
+            </div>
+            <div class="brands">
+                <img src="images/mozilla.png" alt=" Logo Mozilla">
+                <img src="images/googlekorea.png" alt=" Logo Googlekorea">
+                <img src="images/naver.png" alt="Logo Naver">
+                <img src="images/daumkakaologo.png" alt=" Logo Daumkakaologo">
+                <img src="images/airbnb.png" alt="Logo Airbnb">
+            </div>
+        </div>
+    </section>
+
+    <!-- footer -->
+
+    <footer>
+        <div class="foot-items"></div>
+            <img src="images/logo.png" class="blogo" alt="logo">
+        <div class="foot-content">
+            <p>2023 ccglobal all Rights Reserved.</p>
+        </div>
+    </div>
+    </footer>
```

Comment:
- [ ] Kindly fix the footer the space between the image and the text is too much

- [ ] Your design
<img width="1435" alt="Screenshot 2023-03-16 at 10 39 32" src="https://user-images.githubusercontent.com/81444338/225708559-e606271c-8823-43d7-b425-91da1ba043b8.png">

- [ ] Figma Design
<img width="1104" alt="Screenshot 2023-03-16 at 10 42 13" src="https://user-images.githubusercontent.com/81444338/225708333-505ffb9a-ec8c-4dd7-a1b4-c25f8e837adc.png">